SETUP_SUBMITTER_CMF.md

```diff
@@ -35,12 +35,13 @@ Deadline Cloud Monitor is used to both manage your credentials for submitting jo
 
 ## Deadline Software Installation
 
+- clone or download deadline-cloud-for-unreal-engine either from the release branch or mainline depending on whether you'd like the most recent tested release or all of the most recent commits.
+
 ```
-python -m pip install deadline-cloud-for-unreal-engine
+git clone https://github.com/aws-deadline/deadline-cloud-for-unreal-engine.git
```

Comment:
Is `cd deadline-cloud-for-unreal-engine.git` needed here?

Reply:
It totally is yep!